recaptcha_enterprise/demosite/app/templates/comment.html

```diff
@@ -15,73 +15,101 @@
 
 <!DOCTYPE html>
 <html lang="en-us">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <link rel="icon" href="../static/images/icons/favicon.ico" sizes="any" />
-    <link
-      rel="icon"
-      href="../static/images/icons/favicon.svg"
-      type="image/svg+xml"
-    />
-    <link rel="manifest" href="../static/manifest.webmanifest" />
-    <meta name="msapplication-config" content="../static/browserconfig.xml" />
-    <link
-      rel="shortcut icon"
-      sizes="196x196"
-      href="../static/images/icons/favicon-196px.png"
-    />
-    <link
-      rel="apple-touch-icon"
-      href="../static/images/icons/favicon-120px.png"
-      sizes="120x120"
-    />
-    <link
-      rel="apple-touch-icon"
-      href="../static/images/icons/favicon-152px.png"
-      sizes="152x152"
-    />
-    <link
-      rel="apple-touch-icon"
-      href="../static/images/icons/favicon-180px.png"
-      sizes="180x180"
-    />
-    <title>Comments: reCAPTCHA Demo</title>
-    <link
+<head>
+  <meta charset="UTF-8" />
+  <title>Comment: reCAPTCHA Demo</title>
+  <!-- ATTENTION: reCAPTCHA Example (Client Part 1 of 2) Starts -->
+  <!-- See: https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#user-action -->
+  <script src="https://www.google.com/recaptcha/enterprise.js?render={{site_key}}"></script>
+  <script type="text/javascript">
+    function handleClick(event) {
+      event.preventDefault();
+      grecaptcha.enterprise.ready(async () => {
+        const token = await grecaptcha.enterprise.execute(
+            "{{site_key}}",
+            {
+              action: "send_comment",
+            }
+        );
+        await sendComment({ token });
+      });
+    }
+  </script>
+  <!-- See: https://cloud.google.com/recaptcha-enterprise/docs/create-assessment -->
+  <script type="text/javascript">
+    async function sendComment({ comment = "", token }) {
+      // Include the token for server-side assessment.
+      // Include the user input values from your form to pass to the backend.
+      const body = {
+        comment,
+        token,
+      };
+      // Code for fetching the assessment from server-side goes here.
+      // Refer to demo app backend code for more information.
+      // If you already use a library or framework for event handlers, check if this
+      // call can be accommodated the usual way.
```

Comment:
I don't think "call can be accommodated" are the right words?

Reply:
I would suggest using active voice rather than passive voice. I previously had this, "If using a library or framework, can use event handlers its way" to avoid extra words 

"If you are using a library or framework, you can handle events your usual way" ?

Reply:
Done. Thanks
Will modify it in other PRs as well